semantic-analysis/src/Analysis/Effect/Statement.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE KindSignatures #-}
+module Analysis.Effect.Statement
+( -- * Statement effect
+  simport
+, Statement(..)
+) where
+
+import Control.Algebra
+import Data.Kind as K
+import Data.List.NonEmpty (NonEmpty)
+import Data.Text
+
+-- Statement effect
```

Comment:
Can you expand a little on this? Where is this intended to be used, etc?

Reply:
👍🏻